src/main/java/org/verapdf/pd/font/PDType0Font.java

```diff
@@ -18,11 +18,14 @@ public PDType0Font(COSDictionary dictionary) {
 
     public COSDictionary getCIDSystemInfo() {
         if (this.cidSystemInfo == null) {
-            COSDictionary cidFontDict = (COSDictionary)
-                    dictionary.getKey(ASAtom.DESCENDANT_FONTS).get();
+            COSObject cidFontDictObj = dictionary.getKey(ASAtom.DESCENDANT_FONTS).at(0);
+            if (cidFontDictObj == null) {
```

Comment:
You'll receive empty COSObject if the key is absent, so
&& !cidFontDictObj.empty()